app/lua53/lobject.c

```diff
@@ -84,6 +78,8 @@ int luaO_ceillog2 (unsigned int x) {
   x--;
   while (x >= 256) { l += 8; x >>= 8; }
   return l + log_2[x];
+#else
+  return (x == 1) ? 0 : 32 - __builtin_clz(x - 1);
```

Comment:
Is this builtin also available for the ESP8266?  If so then we should be using it rather than a more cludgy `asm` construct.  Where can I get a list of builtins that apply to the ESP32 and ESP8266 compiler?